### PR TITLE
4.0.0 NEWS

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

**Review by hjiebsco — APPROVED** [review]

**Comment by adamdickmeiss** [comment]
And you really read it ??

**Comment by hjiebsco** [comment]
> And you really read it ??

Briefly... Did I miss something big?

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+## 4.0.0 2021-02-26
+
+This releases changes the API for edge-common. There are no known changes
+to the HTTP behavior or the configuration of edge-common.
+
+The existing HTTP-based verticle has been renamed to `EdgeVerticleHttp`.
+A new verticle wihtout a built-in listener is offered, `EdgeVerticleCore`.
```
Comment (hjiebsco) [inline NEWS.md:7]: typo `wihtout`

**Comment by hjiebsco** [comment]
> > And you really read it ??
> 
> Briefly... Did I miss something big?

Read it again. Found a typo :)

**Comment by adamdickmeiss** [comment]
> > > And you really read it ??
> > 
> > 
> > Briefly... Did I miss something big?
> 
> Read it again. Found a typo :)

Damn... Time to rollback..

**Pull request closed without merging**